From d9af6358a40cd76cc64150af31e9897a23ac3faa Mon Sep 17 00:00:00 2001
From: "Dawei.Chien" <dawei.chien@mediatek.com>
Date: Mon, 1 Jun 2015 14:27:16 +0800
Subject: [PATCH 4/7] thermal: add device tree for MTK thermal zone.

Signed-off-by: Dawei Chien <dawei.chien@mediatek.com>

BUG=chrome-os-partner:40082
TEST=build and boot oak-rev1

Change-Id: I762453445e07ef62e612ea0a9eae2472142b00df
---
 arch/arm64/boot/dts/mediatek/mt8173.dtsi | 23 +++++++++++++++++++++++
 1 file changed, 23 insertions(+)

diff --git a/arch/arm64/boot/dts/mediatek/mt8173.dtsi b/arch/arm64/boot/dts/mediatek/mt8173.dtsi
index ecda160..a8cb405 100644
--- a/arch/arm64/boot/dts/mediatek/mt8173.dtsi
+++ b/arch/arm64/boot/dts/mediatek/mt8173.dtsi
@@ -188,6 +188,29 @@
 		clock-output-names = "cpum_ck";
 	};
 
+        thermal-zones {
+                cpu_thermal: cpu_thermal {
+                        polling-delay-passive = <1000>; /* milliseconds */
+                        polling-delay = <0>; /* milliseconds */
+
+                        thermal-sensors = <&thermal 0>;
+
+                        trips {
+                                cpu_passive: cpu_passive {
+                                        temperature = <47000>; /* millicelsius */
+                                        hysteresis = <2000>; /* millicelsius */
+                                        type = "passive";
+                                };
+
+                                cpu_crit {
+                                        temperature = <90000>; /* millicelsius */
+                                        hysteresis = <2000>; /* millicelsius */
+                                        type = "critical";
+                                };
+                        };
+                };
+        };
+
 	timer {
 		compatible = "arm,armv8-timer";
 		interrupt-parent = <&gic>;
-- 
1.9.1

